plugins/modules/network_settings_intent.py

```diff
@@ -804,35 +805,66 @@ def get_network_params(self, site_id):
                 }
             })
 
-        if ntpserver_details is not None:
+        if ntpserver_details.get("value") != []:
             network_settings.update({"ntpServer": ntpserver_details.get("value")})
+        else:
+            network_settings.update({"ntpServer": [""]})
 
         if messageoftheday_details is not None:
             network_settings.update({
                 "messageOfTheday": {
                     "bannerMessage": messageoftheday_details.get("value")[0].get("bannerMessage"),
-                    "retainExistingBanner":
-                    messageoftheday_details.get("value")[0].get("retainExistingBanner"),
                 }
             })
+            if messageoftheday_details.get("value")[0].get("retainExistingBanner") is True:
+                network_settings.get("messageOfTheday").update({
+                    "retainExistingBanner": "true"
+                })
+            else:
+                network_settings.get("messageOfTheday").update({
+                    "retainExistingBanner": "false"
+                })
 
         if network_aaa and network_aaa_pan:
-            network_settings.update({
-                "network_aaa": {
-                    "network": network_aaa.get("value")[0].get("ipAddress"),
-                    "protocol": network_aaa.get("value")[0].get("protocol"),
-                    "ipAddress": network_aaa_pan.get("value")[0]
-                }
-            })
+            if network_aaa_pan.get("value")[0] == "None":
+                network_settings.update({
+                    "network_aaa": {
+                        "network": network_aaa.get("value")[0].get("ipAddress"),
+                        "protocol": network_aaa.get("value")[0].get("protocol"),
+                        "ipAddress": network_aaa2.get("value")[0].get("ipAddress"),
+                        "servers": "AAA"
+                    }
+                })
+            else:
+                network_settings.update({
+                    "network_aaa": {
+                        "network": network_aaa.get("value")[0].get("ipAddress"),
+                        "protocol": network_aaa.get("value")[0].get("protocol"),
+                        "ipAddress": network_aaa_pan.get("value")[0],
+                        "servers": "ISE"
+                    }
+                })
 
         if clientAndEndpoint_aaa and clientAndEndpoint_aaa_pan:
-            network_settings.update({
-                "clientAndEndpoint_aaa": {
-                    "network": clientAndEndpoint_aaa.get("value")[0].get("ipAddress"),
-                    "protocol": clientAndEndpoint_aaa.get("value")[0].get("protocol"),
-                    "ipAddress": clientAndEndpoint_aaa_pan.get("value")[0],
-                }
-            })
+            if clientAndEndpoint_aaa_pan.get("value")[0] == "None":
```

Comment:
Same like above, we are getting the value  clientAndEndpoint_aaa_pan.get("value")[0]  4 times in both if and else part